README.md

```diff
@@ -79,13 +78,23 @@ All metrics are accessible through Prometheus and Grafana dashboards. The gauge
 - `cpumodel` and `gpumodel`, for heterogeneous clusters
 - `deviceid` to select specific GPUs, when available
 
-## Install Autopilot 
+## Install Autopilot
 
 Autopilot can be installed through Helm and need admin privileges to create objects like services, serviceaccounts, namespaces and relevant RBAC.
 
+**NOTE**: this install procedure does NOT allow the use of `--create-namespace` or `--namespace=autopilot` in the `helm` command. This is because out helm chart, creates a namespace with a label, namely, we are creating a namespace with the label `openshift.io/cluster-monitoring: "true"`, so that Prometheus can scrape metrics. This applies to OpenShift clusters **only**. It is not possible, in Helm, to create namespaces with labels or annotations through the `--create-namespace` parameter, so we decided to create the namespace ourselves.
+
+Therefore, we recommend one of the following options, which are mutually exclusive:
+
+- use `--create-namespace` with `--namespace=autopilot` in the helm cli AND disable namespace creation in the helm chart config file `namespace.create=false`. **If on OpenShift**, then manually label the namespace for Prometheus with `label ns autopilot "openshift.io/cluster-monitoring"=`.
```

Comment:
Instead of using bullet notes, can we enumerate like `Option 1` `Option 2` `Option 3`.

Not that it extremely matters, and you already mentioned theres options, but it makes it more clear each bullet is an option itself.

Reply:
Very fair point, change pushed